### PR TITLE
Numerous Bugfixes (see below)

### PR DESCRIPTION
**Pull request by sharmrj**
Merge this after merging #21 
[MWPW-132063](https://jira.corp.adobe.com/browse/MWPW-132063): solved by 340c28704cd0e3f984a35336afc0d418d9ea9859
[MWPW-132115](https://jira.corp.adobe.com/browse/MWPW-132115): Solved by 94f082ef778bf4af14c6b7425bc3af25dd0885af, 21782601dfcc88447789a12eaef121c1a2eaf463
[MWPW-132014](https://jira.corp.adobe.com/browse/MWPW-132014): Solved by 7e5906cf70f9033fe832b8af725a98200ca9acb2
[MWPW-132123](https://jira.corp.adobe.com/browse/MWPW-132123): Solved by b0f25539c07ad8fa7cff81ea9bea5aca96bad325
[MWPW-132124](https://jira.corp.adobe.com/browse/MWPW-132124): Solved by d3da819b9256281d950b1663092dc4c4fea267d4, bf8854f511ac4c2d780c0c013d5d9a4b3c25e63e, 
[MWPW-132129](https://jira.corp.adobe.com/browse/MWPW-132129): Solved by 8a6ac67687db254040dfb4ecf0b5da815035dee5

### CODE REVIEW

**Review by adeshmukh-1585 — DISMISSED** [review]

File: blocks/toc/toc.css [hunk 1]
```diff
@@ -193,9 +198,4 @@
 
 
 @media screen and (max-width: 37.4375rem) {
-  .page-title .toc-mobile-drawer {
-    font-size: 14px;
-    padding: 5px 10px;
-    height: 35px;
-  }
-}
+  .page-title .toc-mobile-dra
```
Comment (SRaj1234) [inline blocks/toc/toc.css]: @sharmrj  Can you check if the media query is needed or not? It looks like incomplete code. Please check.

File: blocks/toc/toc.js [hunk 2]
```diff
@@ -136,7 +136,7 @@ const createMobileTOC = (block) => {
 };
 
 const handleKeyDown = (event) => {
-  event.preventDefault();
+  // event.preventDefault();
```
Comment (SRaj1234) [inline blocks/toc/toc.js]: @sharmrj If this line is not needed, then better to remove it.

**Review by SRaj1234 — CHANGES_REQUESTED** [review]

File: blocks/toc/toc.css (hunk 1 above)
Comment (sharmrj) [reply blocks/toc/toc.css]: good catch. The code was deleted accidentally; I've added it back in.

File: blocks/toc/toc.js (hunk 2 above)
Comment (sharmrj) [reply blocks/toc/toc.js]: I've done so

**Review by nkthakur48 — APPROVED** [review]

**Pull request merged**